### PR TITLE
task/ICS-248 replace public IPFS with dedicated

### PR DESCRIPTION
**Pull request by JGiter**

### CODE REVIEW

File: devops/sandbox/values.yaml [hunk 1]
```diff
@@ -60,6 +60,9 @@ iam-cache-server-helm:
...
       STRATEGY_PRIVATE_KEY: AgBM4BIbZIvZ9ErUDLWmmgw0UMEGk4k9hO54/tos4Rf9bUeewlDyAOrnz+gcl8uXuwFiGazQYNia2JFOjCapKnVt0h3LUjvubN4G2cVoQQl9E2+BDzSb7Ucji2UsdleN1ScOwd6XOADdw7Pa1LUXdpf32rf5YADhdYLGDiYy6hqpJ0OPd+61s92HgtyKHioTSP3fJ4YyFqgXQ4wORRR15rdwJdvzSxUrzGnft/ceuB+ilGJc0qSu/jT+SifQ1411zZ6/vsvhfvSq7q4wXEibn2OWSy7MxxNdYafVC7PbyFgc9i6t8V2YKJzUigIGV53ds9GuMK8i8CeGVg2R3e4iHyuOuWv6LeNjbdUd6QeAynOFVos9rBWwS/L7g+Rv0aUUnD016GYSoPwLGPRwvpyhk3jRtJ94uj4FAkUwWGWKagTA+F0uZyIjq2emO4opE8J5FwbPJOA3SKxfo0yPxgQF7S7uTAGlslQdxSrBk9ICUb5FuMVZ32Bxv3fbJ4R3xIS6D8g9hfPG3/P+zpvTxMzPlOjn7bgXHpbV6qOtt47OjTb9A+ELEVnwF/hOY6efSNa/MYlDU0wjF8CGveKFmYkSo9QHnTx7ruGu1s18yXMQZGK47WEERFR3WS0JqB7iQKv2KhPlMJLQapbROLP3MEYGWpEExDdfQRksmvCpQvQhM/7rtBpjobktxKfP6FVsiCTiL0A8Lkbo+i48oVti3SK6vNF2kDBhy4b6uLWwztYNvz3XL1ZalmcWSwe6iMtscD8Vcjms4FVe3hBRPZbXK5/wOpMq
+      IPFS_CLIENT_PORT: 5001
+      IPFS_CLIENT_PROJECT_SECRET: 9506198a91243f03417aaed60e5eb654
+      IPFS_CLIENT_PROJECT_ID: 2DBFBP5zsjfImw1wBPn1Vx08Yd5
```
Comment (jrhender) [inline devops/sandbox/values.yaml:65]: I think it is better if these go in the `opsValues` for now

File: src/app.module.ts [hunk 2]
```diff
@@ -75,6 +75,35 @@ import { StatusListModule } from './modules/status-list/status-list.module';
...
+          'IPFS_CLIENT_PROJECT_ID'
+        );
+        // https://community.infura.io/t/how-to-add-internet-content-from-a-url-using-ipfs-http-client/5188
+        const auth =
+          'Basic ' +
+          Buffer.from(
+            `${IPFS_CLIENT_PROJECT_ID}:${IPFS_CLIENT_PROJECT_SECRET}`
+          ).toString('base64');
+        return {
+          host: 'ipfs.infura.io',
```
Comment (jrhender) [inline src/app.module.ts:97]: I think this should also be configured and not hardcoded

**Review by jrhender — COMMENTED** [review]
Looks good, some suggestions for improving config

File: devops/sandbox/values.yaml (hunk 1 above)
Comment (JGiter) [reply devops/sandbox/values.yaml:65]: Ok. I didn't know that secrets can go somewhere except `sealedSecret`

File: devops/staging/values.yaml [hunk 3]
```diff
@@ -122,14 +122,17 @@ iam-cache-server-helm:
...
     ENS_URL: https://volta-rpc-vkn5r5zx4ke71f9hcu0c.energyweb.org/
     UNIVERSAL_RESOLVER_URL: https://dev.uniresolver.io/10/identifiers/
     NATS_VERSION: 2.1.8
     NATS_CLIENTS_URL: iam-cache-server-nats.ics.svc.cluster.local:4222
     LOGS_DIRECTORY: /app/logs
     NESTJS_PORT: 3000
     SENTRY_ENV: staging
+    IPFS_CLIENT_PORT: 5001
+    IPFS_CLIENT_PROJECT_SECRET: 9506198a91243f03417aaed60e5eb654
+    IPFS_CLIENT_PROJECT_ID: 2DBFBP5zsjfImw1wBPn1Vx08Yd5
```
Comment (jrhender) [inline devops/staging/values.yaml:134]: I created a new project for staging
```suggestion
    IPFS_CLIENT_PROJECT_SECRET: 6ff4f07ba7643a75d743461b288104c0
    IPFS_CLIENT_PROJECT_ID: 2DDGgPm6nTDVR6HS7o0wjY5ju5j
```

File: devops/prod/values.yaml [hunk 4]
```diff
@@ -119,14 +119,17 @@ iam-cache-server-helm:
...
     ENS_URL: https://rpc.energyweb.org
     UNIVERSAL_RESOLVER_URL: https://dev.uniresolver.io/10/identifiers/
     NATS_VERSION: 2.1.8
     NATS_CLIENTS_URL: iam-cache-server-nats.ics.svc.cluster.local:4222
     LOGS_DIRECTORY: /app/logs
     NESTJS_PORT: 3000
     SENTRY_ENV: prod
+    IPFS_CLIENT_PORT: 5001
+    IPFS_CLIENT_PROJECT_SECRET: 9506198a91243f03417aaed60e5eb654
+    IPFS_CLIENT_PROJECT_ID: 2DBFBP5zsjfImw1wBPn1Vx08Yd5
```
Comment (jrhender) [inline devops/prod/values.yaml:131]: I created a different project for prod
```suggestion
    IPFS_CLIENT_PROJECT_SECRET: 918d9f42f04388d01bbd8913547a60da
    IPFS_CLIENT_PROJECT_ID: 2DDHbWICrMV8MnVjUZZawoyDpDl
```

**Review by jrhender — COMMENTED** [review]
Just some different project values

**Review by jrhender — APPROVED** [review]
Awesome, nice work @JGiter 🚀

**Review by nichonien — APPROVED** [review]

**Pull request merged**